client/me/help/help-courses/index.jsx

```diff
@@ -63,11 +64,13 @@ class Courses extends Component {
 	}
 }
 
-function mapStateToProps( state ) {
+const isWPCOMBusinessPlan = purchase =>
```

Comment:
This looks very useful, I wonder if we could use it in more places. The server uses similar functions for that job - `is_wpcom_business_plan`.

Reply:
That's a great idea, I'll watch out for that in coming PRs and make it reusable on first occasion